audit/projects/k8s-artifacts-prod-bak/buckets/k8s-artifacts-prod-bak/iam.json

```diff
@@ -14,6 +14,12 @@
       ],
       "role": "roles/storage.legacyBucketReader"
     },
+    {
+      "members": [
+        "group:k8s-infra-artifact-admins@kubernetes.io"
+      ],
+      "role": "roles/storage.legacyBucketWriter"
+    },
```

Comment:
Where did this come from? It seems redundant when this group has legacyBucketOwner permissions

Reply:
https://github.com/kubernetes/k8s.io/pull/749 doesn't seem to explicitly set write permission for this user, I'm wondering if this is in-development detritus? Or does assigning the legacyBucketOwner role also assign legacyBucketWriter?